.changeset/chatty-months-report.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/techdocs-common': minor
+'@backstage/plugin-techdocs': minor
```

Comment:
Might also break this out into its own changeset too, since the fixes to `@backstage/plugin-techdocs` are unrelated to OpenStack swift.